.github/workflows/stage-test.yml

```diff
@@ -36,35 +37,37 @@ on:
         required: false
         type: boolean
     secrets:
-      PULUMI_ACCESS_TOKEN:   { required: false }
-      AWS_ACCESS_KEY_ID:     { required: false }
-      AWS_SECRET_ACCESS_KEY: { required: false }
-      AWS_CI_ROLE_ARN:       { required: false }
-      ARM_CLIENT_ID:         { required: false }
-      ARM_CLIENT_SECRET:     { required: false }
-      ARM_SUBSCRIPTION_ID:   { required: false }
-      ARM_TENANT_ID:         { required: false }
-      CODECOV_TOKEN:         { required: false }
-
+      PULUMI_ACCESS_TOKEN: {required: false}
+      AWS_ACCESS_KEY_ID: {required: false}
+      AWS_SECRET_ACCESS_KEY: {required: false}
+      AWS_CI_ROLE_ARN: {required: false}
+      ARM_CLIENT_ID: {required: false}
+      ARM_CLIENT_SECRET: {required: false}
+      ARM_SUBSCRIPTION_ID: {required: false}
+      ARM_TENANT_ID: {required: false}
+      CODECOV_TOKEN: {required: false}
```

Comment:
I think this whole section can now be deleted, instead of just changing the formatting?